aptos-move/framework/aptos-framework/sources/managed_coin.move

```diff
@@ -97,6 +99,32 @@ module aptos_framework::managed_coin {
         coin::register<CoinType>(account);
     }
 
+    /// Destroys capabilities from the account, so that the user no longer has access to mint or burn.
+    public entry fun destroy_caps<CoinType>(account: &signer) acquires Capabilities {
```

Comment:
maybe make an end-to-end with destroy_caps?

Reply:
Added.